Makefile

```diff
@@ -10,3 +10,15 @@ MOCHA_JSX := node_modules/mocha/bin/mocha --compilers jsx:babel-register --recur
 
 test: lint
 	NODE_ENV=test $(MOCHA_JSX) $@
+
+BABEL := node_modules/babel-cli/bin/babel.js
+
+build:
+	@rm -rf dist
+	@echo '✓ Clean out dist directory'
+	@cp -r src dist
+	@echo '✓ Copy /src to /dist'
+	@$(BABEL) dist -d dist > ._babel_tmp && rm ._babel_tmp
+	@echo '✓ Convert ES6 to ES5'
+	@find ./dist -name "*.jsx" | xargs -d"\n" rm
+	@echo '✓ Remove JSX files'
```

Comment:
Can this be simplified to avoid copying the src files over and later deleting them?

```
rm -rf dist && mkdir dist
$(BABEL) src -d dist
```